src/keyboard/Keyboard.tsx

```diff
@@ -514,15 +516,37 @@ export default function Keyboard() {
         )}
       </div>
       {layouts && keymap && behaviors && (
-        <div className="col-start-2 row-start-1 grid items-center justify-center">
+        <div className="col-start-2 row-start-1 grid items-center justify-center relative">
           <KeymapComp
             keymap={keymap}
             layout={layouts[selectedPhysicalLayoutIndex]}
             behaviors={behaviors}
+            scale={keymapScale}
             selectedLayerIndex={selectedLayerIndex}
             selectedKeyPosition={selectedKeyPosition}
             onKeyPositionClicked={setSelectedKeyPosition}
           />
+          <select
```

Comment:
Do we want to use a bare `select` here or used a styled React Aria [Select](https://react-spectrum.adobe.com/react-aria/Select.html) for this?

Reply:
I was following along with the general usage of `select` over the React Aria one. I tried the React Aria one and didn't like the initial results, so if there are future plans to move everything to React Aria's version, there should probably be some time put into a wrapped select with common styling.

Reply:
Fair. WFM.